packages/react-graphql-universal-provider/src/GraphQLUniversalProvider.tsx

```diff
@@ -33,7 +32,7 @@ export function GraphQLUniversalProvider<
 
   const [client, ssrLink] = useLazyRef<
     [
-      import('apollo-client').ApolloClient<any>,
+      import('@apollo/client').ApolloClient<any>,
```

Comment:
The type error was related to a file I missed and didn't update the imports in.

Reply:
My thought was to use this here:
```diff
- import('apollo-client').ApolloClient<any>,
+ ApolloClient<any>,
```
since that type should already be available in this context, does that not work?